phoenix-core/src/main/java/org/apache/phoenix/log/AuditQueryLogger.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.phoenix.log;
+
+import org.apache.phoenix.jdbc.PhoenixConnection;
+import org.apache.phoenix.monitoring.MetricType;
+
+import java.util.Map;
+
+
+/*
+ * Wrapper for query translator
+ */
+public class AuditQueryLogger extends QueryLogger {
+    private LogLevel auditLogLevel;
+
+
+    private AuditQueryLogger(PhoenixConnection connection) {
+        super(connection);
+        auditLogLevel = connection.getAuditLogLevel();
+
+    }
+
+    private AuditQueryLogger() {
+        super();
+        auditLogLevel = LogLevel.OFF;
+    }
+
+    public static final AuditQueryLogger NO_OP_INSTANCE = new AuditQueryLogger() {
```

Comment:
Looks like this mostly duplicates the NO_OP_INSTANCE in QueryLogger. Could clean this up (in follow-up) to not have the same "shell" of a class.